vaadin-date-time-picker-flow-demo/src/main/java/com/vaadin/flow/component/datetimepicker/demo/DateTimePickerView.java

```diff
@@ -36,79 +38,143 @@
-        // source-example-heading: Simple date time picker
+        // source-example-heading: Basic usage
         DateTimePicker dateTimePicker = new DateTimePicker();
 
-        dateTimePicker.addValueChangeListener(
-                event -> updateMessage(message, dateTimePicker));
+        DateTimePicker valueDateTimePicker = new DateTimePicker();
+        LocalDateTime now = LocalDateTime.now();
+        valueDateTimePicker.setValue(now);
         // end-source-example
 
-        dateTimePicker.setId("simple-picker");
-        addCard("Simple date time picker", dateTimePicker, message);
+        dateTimePicker.getStyle().set("margin-right", "5px");
+        div.add(dateTimePicker, valueDateTimePicker);
+        addCard("Basic usage", div);
     }
 
-    private void createLocaleChangeDateTimePicker() {
-        Div message = createMessageDiv("Customize-locale-picker-message");
+    private void valueChangeEvent() {
         // begin-source-example
-        // source-example-heading: Date time picker with different locales
-        // By default, the dateTimePicker uses the current UI locale
+        // source-example-heading: Value change event
         DateTimePicker dateTimePicker = new DateTimePicker();
-        NativeButton locale1 = new NativeButton("Locale: US");
-        NativeButton locale2 = new NativeButton("Locale: UK");
-        NativeButton locale3 = new NativeButton("Locale: CHINA");
 
-        locale1.addClickListener(e -> {
-            dateTimePicker.setLocale(Locale.US);
-            updateMessage(message, dateTimePicker);
-        });
-        locale2.addClickListener(e -> {
-            dateTimePicker.setLocale(Locale.UK);
-            updateMessage(message, dateTimePicker);
+        Div value = new Div();
+        value.setText("Select a value");
+        dateTimePicker.addValueChangeListener(event -> {
+            if (event.getValue() == null) {
+                value.setText("No date time selected");
+            } else {
+                value.setText("Selected date time: " + event.getValue());
+            }
         });
-        locale3.addClickListener(e -> {
-            dateTimePicker.setLocale(Locale.CHINA);
-            updateMessage(message, dateTimePicker);
+        // end-source-example
+
+        Paragraph note = new Paragraph(
+                "Note: The date time picker has a non-empty value only when"
+                        + " both a date and a time have been selected.");
+        VerticalLayout verticalLayout = new VerticalLayout(note, dateTimePicker,
+                value);
+        verticalLayout.setPadding(false);
+        addCard("Value change event", verticalLayout);
+    }
+
+    private void finnishDateTimePicker() {
+        Div message = new Div();
+        /*
+         * TODO: This needs to be extended to be similar to DatePicker
+         * Localizing demo once setI18n() has been implemented
+         */
+        // begin-source-example
+        // source-example-heading: Localizing
```

Comment:
Nit: In case of `datePicker` similar demo is called `Internationalized date picker`. Probably we will need to align now or once setI18n() is implemented

Reply:
Actually in DatePicker it's called the same. These are based on the latest DatePicker demos and I tried to keep this as consistent with it as possible while making adjustments only where it made sense. Note that vaadin.com is not showing the latest demos. See:
https://github.com/vaadin/vaadin-date-picker-flow/blob/master/vaadin-date-picker-flow-demo/src/main/java/com/vaadin/flow/component/datepicker/demo/DatePickerView.java#L269

Imho "Localizing" sounds a bit weird and I'd prefer to use only "Localization" everywhere (now it's "Localization" in the tab, but "Localizing" in the title on the page (I kept consistency with DatePicker for now).

Reply:
Actually looks like vaadin.com is showing the integration test view instead of the demo view. 🙈